clients/venice-admin-tool/src/test/java/com/linkedin/venice/TestAdminTool.java

```diff
@@ -74,13 +74,15 @@ public void testAdminToolRequiresSSLConfigFile() {
   public void testAdminUpdateStoreArg() throws ParseException, IOException {
     final String K1 = "k1", V1 = "v1", K2 = "k2", V2 = "v2", K3 = "k3", V3 = "v3";
     String[] args = { "--update-store", "--url", "http://localhost:7036", "--cluster", "test-cluster", "--store",
-        "testStore", "--rmd-chunking-enabled", "true", "--partitioner-params",
```

Comment:
Is this part of the change PR? Could you update your forked repo and update again?